start.sh

```diff
@@ -11,8 +11,17 @@ python /usr/app/update-display.py
 # Save out the current env to a file so cron job can use it
 export -p > /usr/app/env.sh
 
+# Set default values if these env vars are not set
```

Comment:
Added more complex logic here to support display alternating and more flexible refresh frequency.